.changeset/config.json

```diff
@@ -17,6 +17,7 @@
     "demo-node-esm",
     "demo-typegen",
     "@fuel-ts/docs-snippets",
+    "@fuel-ts/docs-snippets2",
```

Comment:
Why do we prefix the `docs-snippets` with `@fuel-ts/`? I thought we only did this for packages?

Reply:
01c6d10cd371a6d7fafa5f4875943ad3239b8490